torch/utils/_benchmark/utils/valgrind_wrapper/timer_interface.py

```diff
@@ -0,0 +1,351 @@
+import collections
+import dataclasses
+import os
+import re
+import shutil
+import subprocess
+import sys
+import tempfile
+import textwrap
+from typing import List, Tuple
+
+import torch
+
+
+@dataclasses.dataclass(repr=False, eq=False, frozen=True)
+class CallgrindStats(object):
```

Comment:
As a reviewer I think I would definitely appreciate more docstrings on this class.

Reply:
Or even just more types, so I don't have to look in the implementations to figure out what the methods are returning...

Reply:
I have fully type annotated everything now. (Though not pushed yet.) Your suggestion to run everything through MyPy was a good one; thanks! Agree docstrings are also in order.